runfolder/handlers.py

```diff
@@ -51,22 +67,23 @@ def get(self, path):
 
     def post(self, path):
         """
-        Sets the state of the runfolder
+        Sets the state of the runfolder. TODO: Not implemented
         """
-        self.runfolder_svc.set_runfolder_state(path, "TODO")
+        # self.runfolder_svc.set_runfolder_state(path, "TODO")
+        raise NotImplementedError()
 
     @arteria.undocumented
     def put(self, path):
         """
         NOTE: put is provided for test purposes only.
-
-        TODO: Discuss if it should be disabled in production
         """
+        # TODO: Discuss if it should be disabled in production
```

Comment:
In principle I think it should be, but I could also go with being pragmatic and just leave it in for now if it's convenient for testing purposes. Don't know what @johanherman thinks.